extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/AuthenticationProviderService.java

```diff
@@ -72,7 +71,7 @@ public AuthenticatedUser authenticateUser(AuthenticationProvider authenticationP
      *
      * @return
      *     A new UserContext instance for the user identified by the given
-     *     credentials.
+     *     credentials, or null if no such user exists within the database.
```

Comment:
Should we maybe mention what the meaning of returning null is? The method itself says "allow access" and then returns null, but this comment doesn't say anything about returning null implying allowing access.